catalyst/tests/test_finetune2.py

```diff
@@ -30,6 +30,7 @@ def get_loggers(self):
             "csv": dl.CSVLogger(logdir=self._logdir),
             "tensorboard": dl.TensorboardLogger(logdir=self._logdir),
             "mlflow": dl.MLflowLogger(experiment=self._name),
+            "wandb": dl.WandbLogger(project="catalyst_test", config=self.hparams),
```

Comment:
could you please wrap it with `if-else` like `mlflow` one in master branch?

Reply:
Yup done